c++/src/kj/async.c++

```diff
@@ -489,6 +489,13 @@ public:
       }
     }
 #endif
+
+    // Make sure we're not leaking anything from the global freelist either.
```

Comment:
Doh, yes, this appears to be a legit leak.

(In practice `FiberPool`s are typically allocated globally, so the impact is minimal, but it's still definitely a bug.)